docs/demos/mpi.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>I think we could update this intro a little to give more context. E.g.:</p><p> </p><p> </p><p>There are two ways to parallelize the FFT calculations in <code>powerbox</code>. If you have <code>pyfftw</code> installed, you can take advantage of the multithreaded FFT computations that it offers (numpy does not support this), simply by setting <code>nthreads</code> to a number greater than one. However, if you would like to run many FFT's, you may wish to parallelize on a higher level, i.e. run each FFT in a different process (using <code>multiprocessing</code> or <code>mpi</code> or similar). In this case, it is important that the underlying FFT library use only a single thread, or you will get VERY SLOW times because the two layers of threads don't communicate well. In this notebook, we show how to use both multiple threads via <code>pyfftw</code> and also multiple processes with either the <code>numpy</code> or <code>pyfftw</code> backends.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/powerbox/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>